RIOT-OS/RIOT
# boards/olimex-msp430-h2618: add new board

### Contribution description

This adds support of the Olimex MSP430-H2618 board. This board has plenty of RAM and ROM for an MSP430 board (even more than the z1), but still is easily obtainable.


### Testing procedure

1. Check the generated doc
2. Run a few test applications

### Issues/PRs references

- [x] depends on and includes https://github.com/RIOT-OS/RIOT/pull/19705
